examples/seismic/acoustic/acoustic_example.py

```diff
@@ -99,4 +99,5 @@ def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
 
 
 if __name__ == "__main__":
-    run(full_run=True, autotune=False, space_order=6, time_order=2)
+    run(full_run=False, autotune=False, space_order=6, time_order=2, dimensions=(50, 60),
```

Comment:
Why are we changing the way the test is run here? Is this just a debug leftover?